internal/cli/listprofiles/listprofiles.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package listprofiles
+
+import (
+	"fmt"
+
+	"github.com/mongodb/atlas-cli-core/config"
+	"github.com/spf13/cobra"
+)
+
+
+func Builder() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "listprofiles",
```

Comment:
late question, can we make this `profiles list` instead? 
keeping the same format as the cli: https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/cli/teams/list.go